bigquery/google/cloud/bigquery/dbapi/cursor.py

```diff
@@ -154,7 +154,7 @@ def execute(self, operation, parameters=None, job_id=None):
 
         # Wait for the query to finish.
         try:
-            query_job = query_job.result()
+            query_job.result()
         except google.cloud.exceptions.GoogleCloudError:
             raise exceptions.DatabaseError(query_job.errors)
```

Comment:
ISTM this depend on `query_job.result()` rather than `query_job` (since in the above the variable was over-written).

Reply:
Result was returning self, so nothing has changed, really. The errors property should still get populated okay.